config/crd/bases/relocation.openshift.io_imageclusterinstalls.yaml

```diff
@@ -188,20 +184,13 @@ spec:
                     type: string
                 type: object
                 x-kubernetes-map-type: atomic
-              pullSecretRef:
-                description: PullSecretRef is a reference to new cluster-wide pull
-                  secret. If defined, it will replace the secret located at openshift-config/pull-secret.
-                  The type of the secret must be kubernetes.io/dockerconfigjson.
-                properties:
-                  name:
-                    description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
-                      TODO: Add other useful fields. apiVersion, kind, uid?'
-                    type: string
-                type: object
-                x-kubernetes-map-type: atomic
-              release_registry:
+              releaseRegistry:
+                description: 'ReleaseRegistry is the registry of the installed release
+                  image TODO: should this also use the registry of ImageSetRef?'
                 type: string
               version:
+                description: 'Version is the target OCP version for the cluster TODO:
+                  should this use ImageSetRef?'
```

Comment:
Related to this I'm getting an error trying to create the `ImageClusterInstall` in testing saying that this is required.

I don't think we can change that and still conform to the hive `ClusterInstall` API

Reply:
Moving this conversation to the types.go file.